Tasks/google-play-release/GooglePlay.ts

```diff
@@ -778,21 +778,26 @@ function resolveGlobPaths(path: string): string[] {
 
 /**
  * Get obb file for the apk from the parent directory.
- * @returns {string[]} path of the obb file if present else null
+ * @returns {string} path of the obb file if present else null
  */
-function getObbFromParentDirectory(apkPath: string): string {
+function getObbFromParentDirectory(apkPath: string, packageName: string, versionCode: number): string {
     const apkDirectory = path.dirname(apkPath);
-    const parentDirectory = path.dirname(apkDirectory);
-    const filenames = fs.readdirSync(parentDirectory);
-    const obbFile: string | undefined = filenames.find(file => path.extname(file) === '.obb');
+    const filenames = fs.readdirSync(apkDirectory);
+    const expectedMainObbFile: string = 'main.' + versionCode + '.' + packageName + '.obb';
```

Comment:
Yes it should, but in our case we have the necessity to name it the conventional way to automate in the pipeline. Moreover we also felt that its good practice to have it named properly instead of a random name.

Reply:
Got it, thanks! So at the moment expansion files support was introduced with the last release, but there was no requirement for specific file format, so there's no backward support - @leantk do we need to increase major version number then?